### ISSUE TITLE
How to create go_proto_library automatically

### ISSUE THREAD

**Issue by ZhengXinwei-F** [bazelbuild/rules_go#3952]
### What version of rules_go are you using?
`v0.47.0`

### What version of gazelle are you using?

`v0.35.0`

### What version of Bazel are you using?
`6.3.2`

### Does this issue reproduce with the latest releases of all the above?

I don't know.

### What operating system and processor architecture are you using?

```
Linux 30eedc3eab42 5.15.146.1-microsoft-standard-WSL2 #1 SMP Thu Jan 11 04:09:03 UTC 2024 x86_64 x86_64 x86_64 GNU/Linux
```

### What did you do?

I use 'bazel run //:gazelle' to generate BUILD.bazel. 

### What did you expect to see?

Create `go_proto_library`.

### What did you see instead?
I only found `proto_library`. I looked up some information and couldn't locate a solution.

**Comment by ZhengXinwei-F** [comment]
Should I inquire in the gazelle community?

I am a newbie in bazel; thank you for your help.  :-)

**Comment by popovicu** [comment]
Maybe something like [this](https://github.com/popovicu/go-grpc-bazel-example/blob/main/proto/BUILD)? Only instead of the gRPC library, you can use a plain proto.

**Comment by alexeagle** [comment]
> Should I inquire in the gazelle community

Yes this is built-in, see `# gazelle:go_generate_proto` in https://github.com/bazelbuild/bazel-gazelle
